config/security.js

```diff
@@ -48,38 +48,5 @@ module.exports = function (app) {
     next();
   });
 
-  // 5. captcha
-  var captchaKeys = {
-    site: process.env.CAPTCHA_SITE || '<captcha-site>',
-    secret: process.env.CAPTCHA_SECRET || '<captcha-secret>',
-  };
-
-  var checkCaptcha = function(req, res, next) {
-    if (req.body && req.body.recaptcha) {
-      request({
-        url: 'https://www.google.com/recaptcha/api/siteverify',
-        method: 'POST',
-        form: {
-          secret: captchaKeys.secret,
-          response: req.body.recaptcha,
-          remoteip: req.ip
-        },
-        json: true
-      }, function(error, response, body) {
-        if (body.success) {
-          limiter.resetIp(req.ip);
-          next();
-        } else {
-          next({
-            code: 'EBADCSRFTOKEN',
-            error: 'Wrong captcha'
-          });
-        }
-      });
-    } else {
-      next();
-    }
-  };
-
   app.use('/api/', csrfProtection, checkCaptcha, limiter);
```

Comment:
You'll want to kill the checkCaptcha here too. Although I guess I was wrong that you can't remove all of the rest of the file.

(We should also nix CSRF, but that doesn't need to be part of this.)